plugins/com.google.cloud.tools.eclipse.appengine.deploy.ui/plugin.xml

```diff
@@ -8,8 +8,16 @@
   <extension point="org.eclipse.ui.propertyPages">
     <page
         id="com.google.cloud.tools.eclipse"
-        name="Google"
+        name="Google Cloud Platform"
         class="com.google.cloud.tools.eclipse.properties.GooglePropertyPage">
+       <enabledWhen> <!-- this will probably be too strict when App Engine Flex becomes supported -->
+        <adapt type="org.eclipse.core.resources.IProject">
```

Comment:
I mentioned to @akerekes in #85 that we should reference an _expression definition_, via the [_org.eclipse.core.expressions.definition_](http://help.eclipse.org/mars/index.jsp?topic=%2Forg.eclipse.platform.doc.isv%2Freference%2Fextension-points%2Forg_eclipse_core_expressions_definitions.html) extension point. Perhaps defined in `com.google.cloud.tools.eclipse.appengine.facets`?